src/java/htsjdk/samtools/util/Locatable.java

```diff
@@ -22,4 +22,63 @@
      * @return 1-based closed-ended position, undefined if getContig() == null
      */
     int getEnd();
+
+
+    /**
+    * @return the 0-based start position (from the GA4GH spec).
+    */
+    default long getGA4GHStart() {return this.getStart() - 1; }
```

Comment:
I really don't like adding this to Locatable, as this (and the matching end method) will clutter up a lot of other interfaces.  And I think it's really confusing to introduce methods that work in a zero-based coordinate system into a codebase that is almost exclusively 1-based.  Is there a real need for this?